debian/postinst

```diff
@@ -124,6 +124,10 @@ configure)
 		update-initramfs -u -t -k "$(uname -r)"
 	fi
 
+  if [[ -f "/var/lib/delphix-platform/export-home" ]]; then
+    chmod 755 "/var/lib/delphix-platform/export-home"
+  fi
```

Comment:
This shouldn't be necessary. Note that there is no equivalent code needed for the other executables delivered by this package. You should be able to set the permissions on the file directly in the repo.